systemd/ceph-rbd-mirror@.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Ceph rbd mirror daemon
+After=network-online.target local-fs.target
+Wants=network-online.target local-fs.target
+
+[Service]
+LimitNOFILE=1048576
+LimitNPROC=1048576
+EnvironmentFile=-/etc/sysconfig/ceph
+Environment=CLUSTER=ceph
+ExecStart=/usr/bin/rbd-mirror -f --cluster ${CLUSTER} --id %i --setuser ceph --setgroup ceph
```

Comment:
here if the cluster name is not ceph, it will read the ceph.conf by default. So, i think you should add -c to point out which conf you want to read.